CONTRIBUTING.md

```diff
@@ -1,19 +1,16 @@
 # Contributing to Apache Tomcat
 
 Firstly, thanks for your interest in contributing! I hope that this will be a
-pleasant experience for you, and that you will return to continue
+pleasant experience for you and that you will return to continue
 contributing.
 
 Please visit our [Get Involved page](https://tomcat.apache.org/getinvolved.html)
 for more information on how to contribute.
 
 ## Code of Conduct
 
-This project and everyone participating in it is governed by the Apache
-software Foundation's
-[Code of Conduct](https://www.apache.org/foundation/policies/conduct.html). By
-participating, you are expected to adhere to this code. If you are aware of
-unacceptable behavior, please visit the
+This project and everyone participating in it are governed by the Apache Software Foundation's
+[Code of Conduct](https://www.apache.org/foundation/policies/conduct.html). By participating, you are expected to adhere to this code. If you are aware of unacceptable behaviour, please visit the
 [Reporting Guidelines page](https://www.apache.org/foundation/policies/conduct.html#reporting-guidelines)
```

Comment:
Please retain the existing 80 character line length limit. It also helps minimise the diff so the changes are easier to review.